nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Cli.Test/EntryPointTests.Discover.cs

```diff
@@ -106,14 +105,13 @@ public async Task WithSolution()
         [Fact]
         public async Task WithProject()
         {
-            var projectPath = "path/to/my.csproj";
             await RunAsync(path =>
                 [
                     "discover",
                     "--repo-root",
                     path,
                     "--workspace",
-                    path,
+                    "path/to",
```

Comment:
It would be useful to also paths with `whitespaces`. Something like `path/to project`

Reply:
Good call, added a separate test for specifically this.